src/main/java/org.badcode/TicTacToe.java

```diff
@@ -0,0 +1,280 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+
+
+   O | O | O
+  -----------
+   O | O | O 
+  -----------
+   O | O | O
+
+
+ */
+
+import java.util.Scanner;
+
+public class TicTacToe {
```

Comment:
Codacy found an issue: [The utility class name 'TicTacToe' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/gh/EddieDemoOrg/petstore/pullRequest?prid=6129301)